content/blog/2024-11-04-tmix-10/index.md

```diff
@@ -0,0 +1,141 @@
++++
+title = "Linebender in October 2024: resvg stewardship"
+authors = ["Daniel McNab"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+This month's update is very significant, featuring several crate releases and the adoption of resvg.
+
+## Resvg
+
+We would like to thank [Yevhenii Reizner (RazrFalcon)](https://github.com/RazrFalcon) for all their work on resvg.
+The project is...
+
+The biggest update from this month is that .
+
+### Relicense
+
+We are in the process of relicensing the resvg crate from their current MPL-2.0.
+This allows us to more easily integrate existing Linebender code, and enables greater compatibility.
+This was inspired by a similar move made by wgpu (TODO: Link).
+This process can be followed at [resvg#838][].
+
+### Plans
+
+We have no immediate plans for changes in resvg or tiny skia.
+We are currently serving.
```

Comment:
Yeah, to be clear I am currently hoping to avoid writing this section myself, because it requires better writing than I can do quickly - I was providing a template which could or could not be followed.